src/MoBi.R/Services/ParameterValuesTask.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Linq;
+using OSPSuite.Core.Domain.Builder;
+
+namespace MoBi.R.Services
+{
+   public interface IParameterValuesTask
+   {
+      void SetParameterValue(ParameterValuesBuildingBlock pvBB, string parameterName, double newValue);
+   }
+
+   public class ParameterValuesTask : IParameterValuesTask
+   {
+      public void SetParameterValue(ParameterValuesBuildingBlock pvBB, string parameterName, double newValue)
+      {
+         var parameter = pvBB.FirstOrDefault(p => p.Name == parameterName);
+         if (parameter != null)
+            parameter.Value = newValue;
+      }
+   }
```

Comment:
What is the use case for this function? Usually a parameter should be set by the **full path** and not just by name